rust/src/lib/net_state.rs

```diff
@@ -2,9 +2,12 @@
 
 #[cfg(not(feature = "gen_conf"))]
 use std::collections::HashMap;
+use std::fmt;
```

Comment:
Since this `std::fmt` only been used once. There is no need to import it.

Reply:
Thanks fixed